docs/pages/develop/user-interface/assets.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Assets
+description: Learn about using local and remote assets.
+---
+
+Files such as images, videos, sounds, database files for SQLite, or fonts are considered **assets**. They are not part of your project's JavaScript but are part of your app.
+
+Your project can use two types of assets: **Local** and **Remote**. This guide will explain how to use both types of assets.
+
+## Local assets
```

Comment:
in the react-native docs these are typically called static assets, and static assets can actually be "local" or "remote". they can be embedded in the app binary, but they will otherwise be served over the network. this is handled automatically in development.

Reply:
Thanks for the context! I'll rephrase this and rest of the document to use "static assets" as the correct terminology.

Reply:
Updated the guide using the above context.